### PR TITLE
cleanup: Remove unused files 1/4

### PR DESCRIPTION
**Pull request by jjoyce0510**
This PR removes files made redundant by removing the Ember application.

## Checklist
- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/linkedin/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable)

### CODE REVIEW

**Review by dexter-mh-lee — COMMENTED** [review]
@jjoyce0510 seems like our smoke tests use /api/v2/graphql endpoint and is failing

File: metadata-ingestion/src/datahub/metadata/schema.avsc [hunk 1]
```diff
@@ -5219,20 +5219,20 @@
             "fields": [
               {
                 "type": [
-                  "null",
-                  "long"
+                  "long",
```
Comment (dexter-mh-lee) [inline metadata-ingestion/src/datahub/metadata/schema.avsc:5222]: Where are all these changes coming from?

Comment (jjoyce0510) [reply metadata-ingestion/src/datahub/metadata/schema.avsc:5222]: what the...

Comment (jjoyce0510) [reply metadata-ingestion/src/datahub/metadata/schema.avsc:5222]: these recently got in when I merged with master. something is off.

Comment (aseembansal-gogo) [reply metadata-ingestion/src/datahub/metadata/schema.avsc:5222]: https://github.com/linkedin/datahub/pull/3035#issuecomment-893320703

**Review by dexter-mh-lee — APPROVED** [review]
LGTM!

**Review by shirshanka — APPROVED** [review]
LGTM

**Pull request merged**